pg_mooncake/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+TRIES=3
+CONNECTION=postgres://postgres:pg_mooncake@localhost:5432/postgres
+
+cat queries.sql | while read query; do
+    echo "$query"
+    (
+        echo '\timing'
+        yes "$query" | head -n $TRIES
```

Comment:
As per the [benchmark rules](https://github.com/ClickHouse/ClickBench), `[...] the first query should be run after dropping the page cache [...]`. Please grep for `/proc/sys/vm/drop_caches` in the ClickBench repository for examples how to do that.

Reply:
Let's address this ^^ as well.